Documentation/Microsoft.Support.md

```diff
@@ -0,0 +1,12 @@
+# Microsoft Support
+
+This document details Microsoft Support pollicy for CoreWCF.
+
+Runtime Version | Support dependency duration |
+--- | --- |
+.NET Framework 4.x | The specified version of .NET Framework and ASP.NET Core 2.1 |
+Net Core 3.1 | .NET 3.1 LTS - DEcembmer 3, 2022 |
```

Comment:
December has weird casing and incorrect spelling.

Reply:
Thanks for catching that!